pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -822,6 +822,8 @@ func (m *migrationMonitor) startMonitor() {
 	}
 	defer dom.Free()
 
+	logInterval := 0
+
 	for {
 		time.Sleep(400 * time.Millisecond)
```

Comment:
Can we use a constant for both the sleep period and the time interval so that we keep the ratio constant if we happen to change the sleep time?
```go
const sleepPeriodMS = 400
const logPeriodMS = 2000
const logInterval = logPeriodMS / sleepPeriodMS
...
if logCounter % logInterval == 0 {
}
```